### ISSUE TITLE
"SQLite Error 1: 'no such column" exception with EFc5

### ISSUE THREAD

**Issue by pas059** [dotnet/efcore#24940]
Hello,

while upgrading an app from EFc 3.1 to EFc 5, i get a "SQLite Error 1: 'no such column:..." exception when trying the folowing query:
```
var foundPatients = Patients.Include( p => p.SubjectCategory )
							.Select( p => new
							{
								p.FirstName,
								p.LastName,
								p.BirthDate,
								p.SubjectCategory.CategoryName,
								p.Address.Country,
								p.Tests.Count,
							} );
System.Diagnostics.Debug.WriteLine( "Query: " + foundPatients.ToQueryString() );
foreach ( var patientInfo in foundPatients )
{
//....
}
```

where Patients is a DbSet of Patient entity. The complete exception message is:
SQLite Error 1: 'no such column: p.ProfessionProfessionId'.
and  'p.ProfessionProfessionId' refers to a navigation property of Patient to a Profession entity. This code works fine with EFc 3.1 , except the call to ToQueryString() that i added in the EFc5 version for trying to understand and that gives:
```
SELECT "p"."PatientId", "p"."BirthDate", "p"."BirthPlace", "p"."CreationDate", "p"."CreatorId", "p"."Gender", "p"."Mail", "p"."Phone", "p"."ProfessionProfessionId", "p"."ShoeSizeShoeSizeId", "p"."Size", "p"."SubjectCategoryCategoryName", "p"."Weight", "p"."Address_City", "p"."Address_Country", "p"."Address_Description", "p"."Address_Line", "p"."Address_Postcode", "p"."Address_State", "p"."Address_Type", "p"."Person_Comment", "p"."Person_FirstName", "p"."Person_Identifier", "p"."Person_LastName", "p"."Person_Portrait", "s"."CategoryName", "s"."Description", "s"."Group", "s"."SubGroup", (
    SELECT COUNT(*)
    FROM "Tests" AS "t"
    WHERE "p"."PatientId" = "t"."PatientPatientId")
FROM "Patients" AS "p"
INNER JOIN "SubjectCategories" AS "s" ON "p"."SubjectCategoryCategoryName" = "s"."CategoryName"
```
in addition to the exception, what is wondered me is that the query SELECT all the properties of the Patient entity!

Someone can help?
regards
Pascal

### provider and version information

EF Core version: 5.0.6
Database provider: Sqlite
Target framework: (e.g. .NET 5.0)
Operating system:W10
IDE: (e.g. Visual Studio 2019CE 16.9.6

**Comment by AndriySvyryd** [comment]
@pas059 Please provide a small repro project that shows the issue.

**Comment by pas059** [comment]
Hi,
I had not indicated that i use a model designer/code generator tool, and i strongly think that the problems comes from the automatic code generation part; i'm waiting for confirmation from this side and return here back after.
regards

**Comment by ajcvickers** [comment]
**EF Team Triage:** Closing this issue as the requested additional details have not been provided and we have been unable to reproduce it.

*BTW this is a canned response and may have info or details that do not directly apply to this particular issue. While we'd like to spend the time to uniquely address every incoming issue, we get a lot traffic on the EF projects and that is not practical. To ensure we maximize the time we have to work on fixing bugs, implementing new features, etc. we use canned responses for common triage decisions.*

**Comment by maxima120** [comment]
I am having the same issue. There was a navigation property in EF dbContext classes and an FK in SQLite. I removed it by hand and now a Select from the table tries to access that "phantom" property with made up name (eg UserPayment table was linked to UserData table as many to 1 but later that link was removed and it tries to use property UserDataUserId which is obv generated as it doesnt exists anywhere in file system).. I am going to try and repro it in a small project and open another ticket as this is a problem for me.
